components/form/LabeledInput.vue

```diff
@@ -6,6 +6,7 @@ import LabeledTooltip from '@/components/form/LabeledTooltip';
 import { escapeHtml } from '@/utils/string';
 import cronstrue from 'cronstrue';
 import { isValidCron } from 'cron-validator';
+import { debounce } from 'lodash';
```

Comment:
quick change: you can import only the debounce method from lodash, such as:
`import debounce from 'lodash/debounce';`

Reply:
Destructured imports are automatically threeshaked by Webpack

Reply:
Official docs Webpack: https://webpack.js.org/guides/tree-shaking/
Article from Google (more digestible): https://web.dev/reduce-javascript-payloads-with-tree-shaking/